gapic/templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -642,10 +642,12 @@ def test_{{ method_name }}_rest_error():
     )
     # Since a `google.api.http` annotation is required for using a rest transport
     # method, this should error.
-    with pytest.raises(RuntimeError) as runtime_error:
+    with pytest.raises(NotImplementedError) as not_implemented_error:
         client.{{ method_name }}({})
-    assert ("Cannot define a method without a valid 'google.api.http' annotation."
-        in str(runtime_error.value))
+    assert (
+        "Method {{ method.name }} is not available over REST transport"
+        in str(not_implemented_error.value)
+    )
```

Comment:
Shouldn't this be indented forward - i.e. same indent as `client.{{ method_name }}` ?

Reply:
I would have thought so too, but the `not_implemented_error` remains in scope after the `with`, and the `with` catches the exception. See also https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions